package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "eslint-config-opentable",
-  "version": "8.0.0",
+  "version": "9.0.0",
```

Comment:
We don't update the version until the publish. The publish command will up the version and we just save it after

Reply:
👍 will revert back the change. It seemed a version bump was done [here](https://github.com/opentable/eslint-config-opentable/pull/10/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R3), so I assumed I needed a manual version bump 🙂